ci/setup.sh

```diff
@@ -41,6 +43,12 @@ fi
 # Install MySQL 8.0 if DB=mysql80
 if [[ -n ${DB-} && x$DB =~ ^xmysql80 ]]; then
   sudo bash ci/mysql80.sh
+  if [[ -n ${DB_CLIENT-} && x$DB_CLIENT =~ ^xmariadb ]]; then
+    sudo apt-get install -qq libmariadb-dev-compat
+  else
+    sudo apt-get install -qq libmysqlclient-dev
+  fi
+
```

Comment:
I moved the logic to install libmysqlclient-dev or libmariadb-dev-compat from `mysql80.sh` to here. Because the environment variable `DB_CLIENT` is not available in the `sudo bash`.